languages/elixir/exercises/concept/charlists/.docs/after.md

```diff
@@ -0,0 +1,162 @@
+Charlists are created using single quotes.
+
+```elixir
+'hello'
+```
+
+A charlist is a list of integers where the integers represent the Unicode values of a given character — also known as code points.
+
+## Charlists _are_ lists
+
+```elixir
+'' === []
+# => true
+
+is_list('hello')
+# => true
+```
+
+Because charlist are list, you can work with them just like with any other list - using recursion and pattern matching, or using the [`List`][list] module.
```

Comment:
```suggestion
Because charlist are lists, you can work with them just like with any other list - using recursion and pattern matching, or using the [`List`][list] module.
```